### PR TITLE
Update auto-failover-group-sql-mi.md

### PR DESCRIPTION
**Pull request by ruimaiaMSFT**
added information on Scale instances section: recommendations are also enforced when scaling storage (even without changing tier and vCores).

### CODE REVIEW

**Comment by Court72** [comment]
@Stralle

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: azure-sql/managed-instance/auto-failover-group-sql-mi.md [hunk 1]
```diff
@@ -218,7 +218,9 @@ Instances in a failover group remain separate Azure resources, and no changes ma
...
 -->
 
-You can scale up or scale down the primary and secondary instance to a different compute size within the same service tier or to a different service tier. When scaling up within the same service tier, we recommend that you scale up the geo-secondary first, and then scale up the primary. When scaling down within the same service tier, reverse the order: scale down the primary first, and then scale down the secondary. When you scale instance to a different service tier, this recommendation is enforced.
+You can scale up or scale down the primary and secondary instance to a different compute size within the same service tier or to a different service tier. When scaling up within the same service tier, we recommend that you scale up the geo-secondary first, and then scale up the primary. When scaling down within the same service tier, reverse the order: scale down the primary first, and then scale down the secondary. When you scale instance to a different service tier, this recommendation is enforced. The sequence of operations is enforced when scaling the service tier and vcores, as well as storage.
```
Comment (rwestMSFT) [inline azure-sql/managed-instance/auto-failover-group-sql-mi.md]:
```suggestion
You can scale up or scale down the primary and secondary instance to a different compute size within the same service tier or to a different service tier. When scaling up within the same service tier, we recommend that you scale up the geo-secondary first, and then scale up the primary. When scaling down within the same service tier, reverse the order: scale down the primary first, and then scale down the secondary. When you scale instance to a different service tier, this recommendation is enforced. The sequence of operations is enforced when scaling the service tier and vCores, as well as storage.
```

File: azure-sql/managed-instance/auto-failover-group-sql-mi.md [hunk 2]
```diff
@@ -218,7 +218,9 @@ Instances in a failover group remain separate Azure resources, and no changes ma
...
+
```
Comment (rwestMSFT) [inline azure-sql/managed-instance/auto-failover-group-sql-mi.md]:
```suggestion
```

File: azure-sql/managed-instance/auto-failover-group-sql-mi.md [hunk 3]
```diff
@@ -218,7 +218,9 @@ Instances in a failover group remain separate Azure resources, and no changes ma
...
+
```
Comment (rwestMSFT) [inline azure-sql/managed-instance/auto-failover-group-sql-mi.md]:
```suggestion
```

**Pull request merged**